EngineLayer/Mods/Mods.txt

```diff
@@ -96,11 +96,19 @@ MT   Mod
 CF   C1 O2
 //
 ID   Phospho
-TG   S or T or Y
+TG   S or T
+PP   Anywhere.
+NL   H0 or H3 O4 P1
+MT   Mod
+CF   H1 O3 P1
+//
+ID   Phospho
+TG   Y
 PP   Anywhere.
 NL   H0 or H3 O4 P1
 MT   Mod
 CF   H1 O3 P1
+DI   216.0420
```

Comment:
I think these neutral losses and diagnostic ions are associated with particular fragmentation methods right? HCD/CID?